### PR TITLE
Create a proper package

### LINKED ISSUE #38: Packaging
This should be a proper package. Toplevel "errors" and "enums" modules make no sense whatsoever.

### PR DESCRIPTION
**Pull request by smurfix**
Four top-level modules, two of them with generic names, is bad.

Closes #38.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,16 +21,15 @@
         'Programming Language :: Python :: 3',
     ],
     keywords='python3 ping icmp socket tool',
-    packages=setuptools.find_packages(exclude=['contrib', 'docs', 'tests']),
-    py_modules=["ping3", "command_line_ping3", "errors", "enums"],
+    packages=["ping3"],
     python_requires='>=3',
-    install_requires=[],
+    install_requires=["anyio >= 3"],
```
Comment (kyan001) [inline setup.py:26]: why ping3 need `anyio` as `install_requires`?

Comment (smurfix) [reply setup.py:26]: Oops, sorry, that is a misplaced commit from my work to anyio-ize ping3. Will fix shortly.

File: tests/test_command_line_ping3.py [hunk 2]
```diff
@@ -7,8 +7,8 @@
 from unittest.mock import patch
 
 sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-import command_line_ping3  # noqa: linter (pycodestyle) should not lint this line.
-import errors  # noqa: linter (pycodestyle) should not lint this line.
+from ping3 import _main as command_line_ping3  # noqa: linter (pycodestyle) should not lint this line.
```
Comment (kyan001) [inline tests/test_command_line_ping3.py:10]: Can we leave the filename of this one "command_line_ping3.py"? So we can just `from ping3 import command_line_ping3` and it looks more reasonable than `_main` for me.

Comment (smurfix) [reply tests/test_command_line_ping3.py:10]: Fair enough. How about renaming it to "from ping3 import command_line" instead?

Comment (kyan001) [reply tests/test_command_line_ping3.py:10]: Sounds great!

File: ping3/command_line.py [hunk 3]
```diff
@@ -30,3 +30,6 @@ def main(assigned_args: list = None):
 
     for addr in args.dest_addr:
         ping3.verbose_ping(addr, count=args.count, ttl=args.ttl, timeout=args.timeout, size=args.size, interval=args.interval, interface=args.interface)
+
+if __name__ == "__main__":
```
Comment (kyan001) [inline ping3/command_line.py:34]: We suppose to use `python3 -m ping3.command_line ...` to run this line, right?

Should we need a `__main__.py` which has the following content?
```py
from . import command_line

if __name__ == "__main__":
    command_line.main()
```

**Comment by smurfix** [comment]
Thank you.

**Pull request merged**